templates/install-wp-tests.sh

```diff
@@ -147,8 +147,19 @@ install_db() {
 	fi
 
 	# create database
-	mysqladmin drop $DB_NAME -f --user="$DB_USER" --password="$DB_PASS"$EXTRA > /dev/null 2>&1
-	mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
+	if [ $(mysql -e 'show databases;' | grep ^$DB_NAME$) ]
+	then
+		echo "Reinstalling will delete the existing test database ($DB_NAME)"
+		read -p 'Are you sure you want to proceed? [No/Yes]: ' DELETE_EXISTING_DB
+		# Matches on Y, y, Ye, YE, ye, yE, Ys, YS, ys, yS, Yes, yes, YEs, YES, yES, yeS, yEs, yeS, and maybe others
+		if [[ $DELETE_EXISTING_DB =~ [yY]+[eEsS]* ]]
```

Comment:
You should use a more strict regex and set the shell option `nocasematch` for this check.

Here's what the regex could look:
```suggestion
		if [[ $DELETE_EXISTING_DB =~ ^(y|yes)$ ]]
```

The check should then be surrounded in:
```
# Make checks case-insensitive.
shopt -s nocasematch

# [The check and delete operation...]

# Make checks case-sensitive again.
shopt -u nocasematch

Reply:
What's your opinion on the prompt? I'm never really sure what to use. I'm leaving it as [No/Yes] for now but if you know of a better practice, love to hear it. Committing and pushing shortly.

Reply:
Just realized I hadn't adapted the tests. Looking into that now.

Reply:
I usually see `[y/N]` here, which is commonly understood as "yes or no, the default is No".